assets/prototype/infrastructure/TopLevelProviders/Apollo.js

```diff
@@ -0,0 +1,27 @@
+import { ApolloClient } from 'apollo-client';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { HttpLink } from 'apollo-link-http';
+import { ApolloProvider } from '@apollo/react-hooks';
+import get from 'lodash/get';
+
+const { graphqlEndpoint } = window;
+const nonce = get( window, [ 'eejsdata', 'data', 'eejs_api_nonce' ] );
```

Comment:
It will return `undefined`, in case object's nested shape might get out-of-shape.

https://lodash.com/docs/4.17.15#get